tools/packaging/rpm/Dockerfile.build

```diff
@@ -1,22 +1,36 @@
 FROM centos:7
 
-RUN yum upgrade -y && \
-    yum install -y epel-release centos-release-scl && \
-    yum install -y fedpkg sudo make which cmake3 rh-git218 \
-                   automake autoconf autogen libtool ninja-build \
-                   llvm-toolset-7 devtoolset-7-libatomic-devel && \
+RUN yum install -y centos-release-scl epel-release && \
+    yum update -y && \
```

Comment:
I think you should keep `yum upgrade`, to get latest updates into the image.

Reply:
yum **upgrade** and yum **update** will perform the same function that update to the latest current version of package. But the difference is **upgrade** will delete obsolete packages, while **update** will preserve them.

I chose to use **update** because this is what is used by Envoy build on CentOS7 - https://github.com/envoyproxy/envoy-build-tools/blob/master/build_container/build_container_centos.sh

@jwendell what do you think is better here ?

Reply:
I'm fine with update.